### ISSUE TITLE
mouse_x=/mouse_y= still don't work on Windows

### ISSUE THREAD

**Issue by jlnr** [gosu/gosu#293]
See: https://www.libgosu.org/cgi-bin/mwf/topic_show.pl?pid=8519#pid8519

```ruby
require 'gosu'

class Window < Gosu::Window
  def initialize
    super(640, 480, false)
  end
  
  def button_down(id)
    exit if id == Gosu::KbEscape
    if id == Gosu::KbSpace
      self.mouse_x = 10 
      self.mouse_y = 10
    end
  end
  
  def needs_cursor?;true;end
end

Window.new.show
```

Works for me on OS X with Gosu 0.10.1 and SDL HEAD (between 2.0.3 and 2.0.4); to do: test on Windows.

**Comment by jlnr** [comment]
I've reproduced the issue

**Comment by jlnr** [comment]
Fix pushed as a Windows-only gem: 0.10.2.pre2
